test/pantry-test.js

```diff
@@ -0,0 +1,4 @@
+const chai = require('chai');
+const expect = chai.expect;
```

Comment:
you can replace these two lines with 
import { expect } from 'chai';

Reply:
I have also removed this :)

Reply:
ah okay!